docs/protocol/src/protocol/action_descriptions/swap.md

```diff
@@ -40,9 +40,9 @@ using the above witnessed values and where `ds` is a constant domain separator:
 
 The zk-SNARK certifies that the public input fee commitment $cv_f$ was derived from the witnessed values as:
 
-$cv_f = [v_f] G_{v_f} + [\tilde v_f] G_{\tilde v}$
+$cv_f = [-v_f] G_{v_f} + [\widetilde{v_f}] G_{\widetilde{v}}$
```

Comment:
The value of the fee $v_f$ is first negated: https://github.com/penumbra-zone/penumbra/blob/main/crates/core/component/fee/src/fee.rs#L33